src/SimpleFeatureToggle/SimpleFeatureToggle.cs

```diff
@@ -1,41 +1,25 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Diagnostics;
+﻿using System.Collections.Generic;
 using System.Linq;
-using Newtonsoft.Json;
 
 namespace SimpleFeatureToggle
 {
     public class SimpleFeatureToggle
     {
         private readonly SimpleFeatureToggleConfiguration _configuration;
-        private readonly IFileReader _fileReader;
 
-        public SimpleFeatureToggle(SimpleFeatureToggleConfiguration configuration)
-            : this(configuration, new FileReader())
-        {
-        }
-
-        internal SimpleFeatureToggle(SimpleFeatureToggleConfiguration configuration, IFileReader fileReader)
+        internal SimpleFeatureToggle(SimpleFeatureToggleConfiguration configuration)
         {
             _configuration = configuration;
-            _fileReader = fileReader;
         }
 
         public IEnumerable<Feature> GetFeatures()
         {
-            try
+            if(_configuration.FeatureLoadingStrategy == null)
```

Comment:
Maybe we can add argument validation in the constructor to check that we are actually setting the `FeatureLoadingStrategy`?